confidant/utils/misc.py

```diff
@@ -54,3 +56,32 @@ def utcnow():
     """
     now = datetime.utcnow()
     return now.replace(tzinfo=pytz.utc)
+
+
+def prevent_xss_decorator(func):
+    """
+    Prevents XSS attacks:
+     1. Set content type to be application/json
+     2. Set Content Security Policy (already specified at app level)
+     3. Enable XSS Protection
+     4. Prevent MIME Type Sniffing
+     5. Limit Referrer Information
+    """
+    @wraps(func)
+    def wrapper(*args, **kwargs):
+        # Call the original function to get the response
+        pre_xss_response = func(*args, **kwargs)
+
+        # If the response is not a dict, return it as-is
+        if not isinstance(pre_xss_response, dict):
+            return pre_xss_response
```

Comment:
let's try if we can remove this block. Flask's make_response should be able to handle the case of '{}, status_code', see https://flask.palletsprojects.com/en/3.0.x/api/#flask.make_response